pkg/controller/types/networkpolicy.go

```diff
@@ -96,4 +97,10 @@ type NetworkPolicy struct {
 	// AppliedToPerRule tracks if appliedTo is set per rule basis rather than in policy spec.
 	// Must be false for K8s NetworkPolicy.
 	AppliedToPerRule bool
+	// PerNamespaceSelectors maintains a list of unique Namespace selectors of appliedTo groups
+	// of the NetworkPolicy, for which a per-namespace rule is created.
+	// It is used as an index so that namespace updates can trigger corresponding rules
+	// to re-calculate affected namespaces.
+	// This can be non-empty only for NetworkPolicy created for Antrea ClusterNetworkPolicy.
```

Comment:
How about: It is set only for an AntreaClusterNetworkPolicy?

Reply:
Done